javascript/ql/lib/semmle/javascript/frameworks/Apollo.qll

```diff
@@ -0,0 +1,36 @@
+/**
+ * Provides classes for working with Apollo GraphQL connectors.
+ */
+
+import javascript
```

Comment:
This file is only used within your experimental query, so it should probably be moved to the same folder.

Reply:
done 👍